Sources_v3/StreamChat/APIClient/Endpoints/Payloads/MessagePayloads.swift

```diff
@@ -70,7 +70,11 @@ struct MessagePayload<ExtraData: ExtraDataTypes>: Decodable {
         mentionedUsers = try container.decode([UserPayload<ExtraData.User>].self, forKey: .mentionedUsers)
         replyCount = try container.decode(Int.self, forKey: .replyCount)
         
-        reactionScores = try container.decodeIfPresent([String: Int].self, forKey: .reactionScores) ?? [:]
+        latestReactions = try container.decode([MessageReactionPayload<ExtraData>].self, forKey: .latestReactions)
+        ownReactions = try container.decode([MessageReactionPayload<ExtraData>].self, forKey: .ownReactions)
+        reactionScores = try container
+            .decodeIfPresent([String: Int].self, forKey: .reactionScores)?
+            .reduce(into: [:]) { $0[.init(rawValue: $1.key)] = $1.value } ?? [:]
```

Comment:
`[ReactionType: Int].self` doesn't work?

Reply:
Yep, doesn't work

Reply:
How come? It should work IMO. Maybe `ReactionType` is not encoded properly using `singleValueContainer`?

Reply:
@evsaev Probably the generated `Codable` conformance encodes it as `{ "rawValue": "😱" }` instead of `"😱"`